.github/workflows/lighthouse-ci.yml

```diff
@@ -0,0 +1,71 @@
+name: Google Lighthouse
+
+on:
+  issue_comment:
+    types: [edited]
+
+jobs:
+  generate_lighthouse_audit:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add comment to PR
+        id: loading_comment_to_pr
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          GITHUB_TOKEN: ${{ secrets.LIGHTHOUSE_APP_TOKEN }}
```

Comment:
This should not be set. By default it will use `GITHUB_TOKEN` which is already an Environment Variable